src/zkevm_specs/opcode/bitwise.py

```diff
@@ -0,0 +1,60 @@
+from typing import Sequence
+from ..encoding import U8, is_circuit_code
+
+@is_circuit_code
+def check_xor(
+        a8s: Sequence[U8],
+        b8s: Sequence[U8],
+        c8s: Sequence[U8]
+):
+    assert len(a8s) == len(b8s) == len(c8s) == 32
+
+    for i in range(0, 32):
+        assert a8s[i] ^ b8s[i] == c8s[i]
+
+
+def test_check_xor():
+    a8s = [1]
+    b8s = [4]
+    c8s = [5]
```

Comment:
`len(a8s)` is 1, how come the test pass for this?

Reply:
fixed